### DISCUSSION TITLE
Brainstorm

### DISCUSSION THREAD

**Discussion by JosePabloGFnl** [Special-Unitary-Group/xolote#10] [Announcements]
**Essentials**

1. Interface.
2. Search engine for students doing research, they upload a set of papers (that they already have) and it will serve as the knowledge base of the LLM
    - Based on our experience (speech)
    - Additionally, a search engine for papers that the researcher did not know
3. Enter a paper and the next step is to enter if you want to Summarize, Explain or Search
    1. Summarizing is one thing, explaining is another. We are not so interested in the bot summarizing something new, but rather in collecting the information we want. It's looking for a ball in a huge toy chest.
4. Being able to summarize the articles in the sections you want, not only giving you the necessary articles for what you are looking for but also explaining why or where you can find what interests you
5. Separate by research categories: Medicine, Computer Sciences, Natural Sciences (this is so that the promt is more specific and avoids tagging too much)
6. The model explains the context of a chain of references, gives you bullet points in a sequence (possible future idea)
7. Add the sources where you found the answers
8. Improve the user prompt internally
    1. https://app.jenni.ai/ Does not answer some questions correctly. I think this may be due to loss of document context, or the user's prompt not being detailed enough for the model.

**Non-essential**

1. Inclusion of metadata such as: Number of citations, years, authors, areas, etc.
2. Quality index to prioritize information sources.

**Comment by netotz** [comment]
There are other AI tools that attempt to chat with PDFs or files in general. However, it seems that they're not optimized to reply with accuracy, e.g. when there is some degree of ambiguity in the user's prompt, the model doesn't ask for a more specific prompt.
